independent-publisher-connectors/Coupa/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4343 @@
+        },
+        "updated-at": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Automatically created by Coupa in the format",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "receipt-date"
+      ]
+    },
+    "LookupValue": {
+      "type": "object",
+      "properties": {
+        "active": {
+          "type": "boolean",
+          "description": "Active\\Inactive"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the lookup value"
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of the lookup value"
+        },
+        "external-ref-num": {
+          "type": "string",
+          "description": "External Ref Num (actual account value when used in accounting)"
+        },
+        "id": {
+          "type": "integer",
+          "description": "Coupa unique identifier",
+          "format": "int32",
+          "readOnly": true
+        },
+        "is-default": {
+          "type": "boolean",
+          "description": "is_default"
+        },
+        "lookup": {
+          "$ref": "#/definitions/Lookup"
+        },
+        "lookup-id": {
+          "type": "integer",
+          "description": "lookup_id",
+          "format": "int32"
+        }
+      },
+      "required": [
+        "lookup",
+        "name"
+      ]
+    },
+    "Lookup": {
+      "type": "object",
+      "properties": {
+        "active": {
+          "type": "boolean",
+          "description": "Active\\Inactive Flag"
+        },
+        "description": {
+          "type": "string",
+          "description": "description"
+        },
+        "id": {
+          "type": "integer",
+          "description": "Coupa unique identifier",
+          "format": "int32",
+          "readOnly": true
+        },
+        "level-1-name": {
+          "type": "string",
+          "description": "level_1_name"
+        },
+        "level-2-name": {
+          "type": "string",
+          "description": "level_2_name"
+        },
+        "level-3-name": {
+          "type": "string",
+          "description": "level_3_name"
+        },
+        "level-4-name": {
+          "type": "string",
+          "description": "level_4_name"
+        },
+        "name": {
+          "type": "string",
+          "description": "name"
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "FormResponse": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer",
+          "description": "Coupa unique identifier",
+          "format": "int32",
+          "readOnly": true
+        },
+        "updated-at": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Automatically created by Coupa in the format YYYY-MM-DDTHH:MM:SS+HH:MMZ",
+          "readOnly": true
+        }
+      }
+    }
+  }
```

Comment:
@NovaGL your swagger file is missing some required properties. please add them even if you have them empty. plesae see below.
```
"parameters": {},
"responses": {},
"securityDefinitions": {},
"security": [],
"tags": []
```

Your swagger file should match with this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json).

Reply:
Apologies I was not clear enouph on this. You had an empty object in the security array, and I was asking to remove it. Please readd the above properties.